src/Plot.tsx

```diff
@@ -95,6 +102,8 @@ export default function Plot(props: PlotProps) {
             height={height}
             style={{ ...defaultSvgStyle, ...svgStyle }}
           >
+            <MarkerDefs />
```

Comment:
If is only used for the arrows, shouldn't be inside de arrow annotation component? or it needs to be in there?

Reply:
It's to avoid repeating the same definitions many times again. It's small and we can optimize and not add it if no annotations are rendered.